src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -43,72 +38,31 @@ class ScalaSettings extends Settings.SettingGroup {
   val nobootcp = BooleanSetting("-nobootcp", "Do not use the boot classpath for the scala jars.")
   val strict = BooleanSetting("-strict", "Use strict type rules, which means some formerly legal code does not typecheck anymore.")
 
-  val argfiles = BooleanSetting("@<file>", "A text file containing compiler arguments (options and source files)")
```

Comment:
OK, so I guess rather than dropping the option we should implement it /
port it from scalac.


On Mon, Mar 14, 2016 at 4:35 PM, Sébastien Doeraene <
notifications@github.com> wrote:

> In src/dotty/tools/dotc/config/ScalaSettings.scala
> <https://github.com/lampepfl/dotty/pull/1177#discussion_r56019985>:
>
> > @@ -43,72 +38,31 @@ class ScalaSettings extends Settings.SettingGroup {
> >    val nobootcp = BooleanSetting("-nobootcp", "Do not use the boot classpath for the scala jars.")
> >    val strict = BooleanSetting("-strict", "Use strict type rules, which means some formerly legal code does not typecheck anymore.")
> >
> > -  val argfiles = BooleanSetting("@<file>", "A text file containing compiler arguments (options and source files)")
>
> This is important for Windows, which has a limit on the command-line
> length. Without this option it is impossible to compile the Scala standard
> library on Windows, for example, because there are too many files to list
> on the command line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lampepfl/dotty/pull/1177/files#r56019985>.
>



-- 
Martin Odersky
EPFL